classes/submission/Sanitizer.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+/**
+ * @file classes/submission/Sanitizer.php
+ *
+ * Copyright (c) 2014-2020 Simon Fraser University
+ * Copyright (c) 2000-2020 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class Sanitizer
+ *
+ * @brief A sanitization class to sanitize submission data before saving
+ */
+
+ namespace PKP\submission;
+
+use Exception;
+use Illuminate\Support\Arr;
+use PKP\core\PKPString;
+
+class Sanitizer 
+{
+    /**
+     * Defined sanitization rule/method mapping to attributes
+     * It's possible to have multiple sanitization rule for a single attributes
+     * 
+     * $sanitizeMap = [
+     *   'attribute_1' => 'class_method_1',
+     *   'attribute_2' => ['class_method_21', 'class_method_22'],
+     *   ...
+     * ]
+     */
+    protected array $sanitizeMap = [];
```

Comment:
Is this ever populated (and used)? `$object->{$methodName}` patterns make me nervous, and one is introduced here, but I don't think there's any use of it to audit.

Reply:
@asmecher it is not but it's introduced there to have a secondary option to apply sanitization rules . May be too much option is not right way to do stuff . but before I remove it , we need to first come to a concrete decision first that if we will use any `sanitization` process or a `validation` process only .